bolt/lib/Profile/YAMLProfileReader.cpp

```diff
@@ -354,7 +357,15 @@ Error YAMLProfileReader::readProfile(BinaryContext &BC) {
       matchProfileToFunction(YamlBF, Function);
   }
 
-  for (auto &[CommonName, LTOProfiles]: LTOCommonNameMap) {
+  using BinaryFunctionProfileVector =
+      std::vector<llvm::yaml::bolt::BinaryFunctionProfile *,
+                  std::allocator<llvm::yaml::bolt::BinaryFunctionProfile *>>;
+
+  llvm::StringMap<BinaryFunctionProfileVector> LTOCommonNameMap;
```

Comment:
This definition of `LTOCommonNameMap` will override the class member.

Reply:
Done